manifests/h/HiroSystems/Clarinet/2.2.1/HiroSystems.Clarinet.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: HiroSystems.Clarinet
+PackageVersion: 2.2.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{94BFBC31-28F8-4BB8-8C71-12B5412DD11D}'
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
 - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 716.)